cluster-autoscaler/cloudprovider/clusterapi/clusterapi_utils_test.go

```diff
@@ -0,0 +1,227 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clusterapi_test
```

Comment:
I don't think I understand why the tests are in a different package here. That's unusual and doesn't seem to be the case elsewhere in this project.

Reply:
Though I wouldn't personally call it a trick. It's very deliberate. This pattern occurs quite often in the go runtime tree:

```console
$ find . -name export_test.go | wc -l
30
```